src/encoded/static/scss/style.scss

```diff
@@ -103,13 +103,15 @@ $brand-info: #4F689E;
         "encoded/modules/award",
         "encoded/modules/pipeline",
         "encoded/modules/quality_metric",
+        "encoded/modules/schema",
         "encoded/modules/tooltip",
         "encoded/modules/badge",
         "encoded/modules/news",
         "encoded/modules/visualize",
         "encoded/modules/home",
         "encoded/modules/blocks",
-        "encoded/modules/layout-editor";
+        "encoded/modules/layout-editor",
+        "encoded/modules/item";
```

Comment:
Again, if we reverted changes to item, is this impacted?  If this needs to be in here should it be in the schemas feature.

Reply:
Looks like Chloe added _items.scss. It didn’t exist before. So that line should be deleted.